app/src/store/playground/playgroundStore.tsx

```diff
@@ -210,6 +211,26 @@ export const createPlaygroundStore = (initialProps: InitialPlaygroundState) => {
         ],
       });
     },
+    updateModelSupportedInvocationParameters: ({
+      instanceId,
+      supportedInvocationParameters,
+    }) => {
+      const instances = get().instances;
+      set({
+        instances: instances.map((instance) => {
+          if (instance.id === instanceId) {
+            return {
+              ...instance,
+              model: {
+                ...instance.model,
+                supportedInvocationParameters,
+              },
+            };
+          }
+          return instance;
+        }),
+      });
+    },
```

Comment:
probably just need to pull any with defaults that don't exist in invocationParameters into invocationParameters here, right? to respect defaults

Reply:
Good idea

Reply:
We now merge defaults in to parameters that do not exist yet, or exist but do not have a value